sale_coupon_auto_refresh/tests/test_sale_coupon_auto_refresh.py

```diff
@@ -0,0 +1,70 @@
+# Copyright 2021 Tecnativa - David Vidal
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+from odoo.tests import Form, common
+
+
+class TestWebsiteSaleCouponAutorefresh(common.SavepointCase):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
```

Comment:
```suggestion
        super().setUpClass()
        cls.env = cls.env(context=dict(cls.env.context, tracking_disable=True))
```
always and especially if you create partnes, orders and whatever has a lot of tracking features ;)